Tools/generate_board_targets_json.py

```diff
@@ -1,31 +1,45 @@
 #!/usr/bin/env python3
-""" Script to generate a JSON config with all build targets (for CI) """
+"""
+Script to generate a JSON config with all build targets (for CI), in 2 different versions:
+
+1. For enumerating targets: {"include":[{"target": "px4_fmu-v5x_default", "container": "nuttx-focal"}, ...]}
+
+Called by : Tools/generate_board_targets_json.py -e
+
+---
+
+2. For board information: {"include":[{"target": "px4_fmu-v5x_default", "board_id": "51"}, ...]}
+
+Called by : Tools/generate_board_targets_json.py -b
```

Comment:
There's pretty much no overlap in functionality between the two, so why not add a separate script?

Reply:
I wanted to keep the 'excluded manufacturers' and 'excluded board types (`test`, etc)' consistent between the two cases, but yes I do feel like there's little overlap :0

Will separate it out!